src/javascripts/components/Chat.js

```diff
@@ -1,19 +1,130 @@
-var React = require('react');
-var ChatBox_settings = require('./ChatBox_settings');
-var ChatBox = require('./ChatBox');
-var ChatInput = require('./ChatInput');
-
+import React from 'react'
+import ChatBoxSettings from './ChatBoxSettings'
+import ChatBox from './ChatBox'
+import ChatInput from './ChatInput'
+import { resMsgs } from '../helper'
 
 class Chat extends React.Component {
+  constructor() {
+      super()
+      this.state = {
+        messages: [
+          {
+            user: "blue",
+            message: "blah blah blah"
+          }, 
+          {
+            user: "yellow",
+            message: "Lorem ipsum dipsum"
+          },
+          {
+            user: "yellow",
+            message: "Hallo lorem"
+          }
+        ]
+      }
+  }  
+
+  componentWillMount() {
+    const localStorageRef = localStorage.getItem(`messages`)  
```

Comment:
you could use a ternary like so:
`const localStorageRef = localStorage ? localStorage.getItem('messages') : null`